src/Makefile

```diff
@@ -112,6 +112,9 @@ CSRC := $(STARTUPSRC) \
         $(SWIFTNAV_ROOT)/src/pps.o \
         $(SWIFTNAV_ROOT)/src/decode.o \
         $(SWIFTNAV_ROOT)/src/signal.o \
+        $(SWIFTNAV_ROOT)/src/ndb.o \
+        $(SWIFTNAV_ROOT)/src/ndb_p1.o \
```

Comment:
In progress. I had to distract to cfs fix and rebasing to Leith's changes. Will do changes now according to these requirements.